easybuild/easyconfigs/t/TensorRT/TensorRT-4.0.1.6-fosscuda-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,36 @@
+name = 'TensorRT'
+version = '4.0.1.6'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://developer.nvidia.com/tensorrt'
+description = """NVIDIA TensorRT is a platform for high-performance deep learning inference"""
+
+sources = ['%(name)s-%(version)s.Ubuntu-16.04.4.x86_64-gnu.cuda-9.2.cudnn7.1.tar.gz']
+checksums = ['fc5fd2ba86225ed62e60870cc24896107770a59829363fcb49229c82bb1f4fb5']
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+
+builddependencies = [
+    ('protobuf', '3.6.1'),
+]
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('cuDNN', '7.1.4.18'),
+]
+
+extract_sources = True
+keepsymlinks = True
+
+exts_list = [
+    ('protobuf-python', '3.6.1', {
+        'modulename': 'google.protobuf',
+        'source_tmpl': 'protobuf-python-%(version)s.tar.gz',
+        'source_urls': ['https://github.com/google/protobuf/releases/download/v%(version)s/'],
+        'start_dir': 'python',
+        'use_pip': False,
```

Comment:
@akesandgren Why not use `pip` here, any specific reason?

Reply:
Copied from the tensorflow easyconfig, so just following your lead :-)

Reply:
Checked it again, there's apparently a good reason for it, it fails with an error when you try to use `pip`:

```
Can't find required file: ../src/google/protobuf/descriptor.proto
```